bascule/basculehttp/log.go

```diff
@@ -0,0 +1,34 @@
+package basculehttp
+
+import (
+	"context"
+	"os"
+
+	"github.com/go-kit/kit/log"
+)
+
+var (
+	defaultLogger = log.NewJSONLogger(log.NewSyncWriter(os.Stdout))
```

Comment:
It's dangerous to have a `defaultLogger` that uses `log.NewSyncWriter(os.Stdout)`.  You can't guarantee that application code isn't also using a separate instance created the same way, and thus you'd have a race condition.  The `defaultLogger` should just be a `NopLogger`, and require application code to explicitly set a logger via your option functions.